### ISSUE TITLE
setup.py "broken": Failed to import KivyUnitTest version for setup.py

### ISSUE THREAD

**Issue by AndreMiras** [KeyWeeUsr/KivyUnitTest#5]
Since 36d542067a5bf93e6c16622959255d1fa2f49783, it doesn't see, possible to install kivyunittest from a clean environment.

```
pip install --upgrade https://github.com/KeyWeeUsr/KivyUnitTest/archive/36d542067a5bf93e6c16622959255d1fa2f49783.zip
Collecting https://github.com/KeyWeeUsr/KivyUnitTest/archive/36d542067a5bf93e6c16622959255d1fa2f49783.zip
  Downloading https://github.com/KeyWeeUsr/KivyUnitTest/archive/36d542067a5bf93e6c16622959255d1fa2f49783.zip
    Complete output from command python setup.py egg_info:
    Failed to import KivyUnitTest version for setup.py

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-6Qcggz-build/
```
Also the case from pypi with 0.1.7 and 0.1.6:
```
pip install kivyunittest==0.1.7
Collecting kivyunittest==0.1.7
  Using cached kivyunittest-0.1.7.zip
    Complete output from command python setup.py egg_info:
    Failed to import KivyUnitTest version for setup.py

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-WTYNU0/kivyunittest/
```

**Comment by KeyWeeUsr** [comment]
Oh my... sorry, I'll look at that soon :)

**Comment by KeyWeeUsr** [comment]
Fixed in https://github.com/KeyWeeUsr/KivyUnitTest/commit/a643cef29f9a7c888bb14fdea577480fe0de2941, available as `0.1.8` on PyPI.
